.github/workflows/build_wheels.yml

```diff
@@ -51,7 +56,7 @@ jobs:
           python -m pip install pep517
           python -m pep517.build -s .
```

Comment:
Running this locally gives

```
pep517.build is deprecated. Consider switching to https://pypi.org/project/build/
```

We can do this instead (need to pip install beforehand)
```
python -m build --sdist
```